pycpfcnpj/calculation.py

```diff
@@ -1,47 +1,62 @@
 DIVISOR = 11
 
+CPF_WEIGHTS = ((10, 9, 8, 7, 6, 5, 4, 3, 2),
+              (11, 10, 9, 8, 7, 6, 5, 4, 3, 2))
+CNPJ_WEIGHTS = ((5, 4, 3, 2, 9, 8, 7, 6, 5, 4, 3, 2),
+               (6, 5, 4, 3, 2, 9, 8, 7, 6, 5, 4, 3, 2))
 
-def first_check_digit(number, weighs):
-    """ This function calculates the first check digit to
-        verify a cpf or cnpj vality.
 
-        :param number: cpf or cnpj number to check the first
-                       digit.  Only numbers.
+
+def calculate_first_digit(number):
```

Comment:
This function ended up returning two possibilities (a string with number and, now, a letter). Try to maintain it predictable and with single responsibility.

Reply:
Fixed in https://github.com/matheuscas/pycpfcnpj/pull/19/commits/4a2080a4989c99070e21298527003fd4958da81d